Modules/_winapi.c

```diff
@@ -3046,99 +3047,44 @@ _winapi.ReportEvent
         The event category.
     event_id: unsigned_int(bitwise=False)
         The event identifier.
-    strings: object(subclass_of='&PyList_Type')
-        A list of strings to be inserted into the event message.
-    raw_data: Py_buffer(accept={str, buffer, NoneType}) = None
-        The raw data for the event.
+    string: object
```

Comment:
If you pass a single string, you can use the LPCWSTR converter.
```suggestion
    string: LPCWSTR
```